### ISSUE TITLE
I should create a file similar to a .env

### ISSUE THREAD

**Issue by Thatguy553** [Thatguy553/MangaWebsiteProject#4]
I have variables with the same information in use all over the website and im sure more to come, I should probably use putenv() and getenv() in php to make my own file like a .env file. Only issue with that is figuring out how to also allow JS to use this file.

**Comment by AgentGeek** [comment]
It's possible to parse the env inside a script tag in views files and call that script variable in the JS files.

Here's a detailed input:
we set an putenv() in the header
we parse the getenv in the views file(s) and parse it into script
```
<script>
var name = '<?php echo getenv();?>';
</script>
```
We call the variable in JS file either with DOM or define variable types.

It's the only thing i can think of right now.

**Comment by Thatguy553** [comment]
> It's possible to parse the env inside a script tag in views files and call that script variable in the JS files.
> 
> Here's a detailed input:
> we set an putenv() in the header
> we parse the getenv in the views file(s) and parse it into script
> 
> ```
> <script>
> var name = '<?php echo getenv();?>';
> </script>
> ```
> 
> We call the variable in JS file either with DOM or define variable types.
> 
> It's the only thing i can think of right now.

Good Idea, its been a few months since i touched this or any project so I need to refresh myself however I will be working on this project again here soon.
